var/spack/repos/builtin/packages/paradiseo/package.py

```diff
@@ -38,28 +39,14 @@ def install(self, spec, prefix):
         options = []
         options.extend(std_cmake_args)
 
-        dsuf = 'dylib' if sys.platform == 'darwin' else 'so'
         options.extend([
-            '-DCMAKE_BUILD_TYPE:STRING=%s' % ('DEBUG' if '+debug' in spec else 'RELEASE'),
+            '-DCMAKE_BUILD_TYPE:STRING=%s' % ('Debug' if '+debug' in spec else 'Release'),
             '-DINSTALL_TYPE:STRING=MIN',
             '-DSMP:BOOL=%s' % ('TRUE' if '+smp' in spec else 'FALSE'), # Note: This requires a C++11 compatible compiler
             '-DEDO:BOOL=%s' % ('TRUE' if '+edo' in spec else 'FALSE'),
             '-DENABLE_CMAKE_TESTING:BOOL=%s' % ('TRUE' if '+tests' in spec else 'FALSE')
         ])
-
-        # MPI
-        if '+mpi' in spec:
-            options.extend([
-                '-DMPI:BOOL=TRUE',
```

Comment:
Yes, of course. I had meant to do that :-/ I'll fix it tomorrow and squash all of the commits.